environments.json

```diff
@@ -2554,15 +2837,7 @@
     "note_id": "harmony-flag",
     "release": "2015-09-08",
     "retire": "2018-04-01",
-    "obsolete": true,
-    "test_suites": [
-      "es5",
-      "es6",
-      "es2016plus",
-      "esnext",
-      "esintl",
-      "non-standard"
-    ]
+    "obsolete": true
```

Comment:
should we change it to `"very"` here?

Reply:
I'm not sure about 0.10+. Some users prefer node.green fork since they can't find the required Node version here.